docker/prepare_proxy.sh

```diff
@@ -41,6 +41,12 @@ if [[ -z "${ISTIO_PROXY_PORT-}" ]] || [[ -z "${ISTIO_PROXY_UID-}" ]]; then
 fi
 
 iptables -t nat -A PREROUTING -p tcp -j REDIRECT --to-port ${ISTIO_PROXY_PORT}
+# To make sure when pod A wants to talk to service A, which is backed by pod A,
+# the traffic is going through proxy twice, client-side and server-side.
+# lo traffic doesn't go through PREROUTING, so needed to be processed in OUTPUT.
```

Comment:
can you expand on the meaning on the rule? e.g. concretely in terms of TCP quadruples and POD_IPs